src/controllers/usersController.ts

```diff
@@ -133,12 +133,17 @@ export const signIn = async (req: Request, res: Response) => {
 
     const sessionId = uuidv4();
 
-    const sessionData = (await SessionsModel.create({
+    const sessionData = await SessionsModel.create({
       sid: sessionId,
       userId: user.id,
-    })) as unknown as Session;
+    });
 
-    res.json({ message: "success", sessionId });
+    const result = {
```

Comment:
The frontend requires the while user, so they store in a state management like redux or context, so you should return to them the whole user as well as the sessionId.